options/posix/include/unistd.h

```diff
@@ -200,7 +200,7 @@ int setegid(gid_t);
 int seteuid(uid_t);
 int setgid(gid_t);
 int setpgid(pid_t, pid_t);
-pid_t setpgrp(pid_t, pid_t);
+int setpgrp(void);
```

Comment:
Maybe we can conform to the POSIX.1 version, which has ``pid_t`` as return type.

Reply:
My man page does not list a `setpgrp` with a `pid_t` return type, I only have these two
```
int setpgrp(void);                   /* System V version */
int setpgrp(pid_t pid, pid_t pgid);  /* BSD version */
```

Reply:
https://linux.die.net/man/2/setpgrp

Reply:
Done as requested.